src/Common/src/Interop/Windows/Kernel32/Interop.GlobalLock.cs

```diff
@@ -10,9 +10,9 @@ internal partial class Interop
     internal partial class Kernel32
     {
         [DllImport(Libraries.Kernel32, PreserveSig = true)]
-        internal static extern IntPtr GlobalLock(IntPtr hMem);
+        internal static extern IntPtr GlobalLock(HandleRef hMem);
 
         [DllImport(Libraries.Kernel32, PreserveSig = true)]
-        internal static extern int GlobalUnlock(IntPtr hMem);
+        internal static extern int GlobalUnlock(HandleRef hMem);
```

Comment:
Our automated testing of System.Management is very limited.  Please don't replace these overloads that it's using.  You can just add the IntPtr overloads here.

Reply:
Is there a difference between IntPtr and HandleRef marshalling? Just curious?

Reply:
Yes, HandleRef does additional stuff related to keeping alive a managed object during the P/Invoke.  It's kind of a precursor to SafeHandle.